src/server/pfs/server/driver.go

```diff
@@ -4652,9 +4654,9 @@ func (d *driver) forEachPutFile(pachClient *client.APIClient, server pfs.API_Put
 			}
 			pr, pw = io.Pipe()
 			pr := pr
-			limiter.Acquire()
+			d.downloadLimiter.Acquire()
```

Comment:
@brycemcanally It seems like we're using this same limiter to restrict the parallelism of load files from the GRPC stream (in addition to requests with a URL that pull from outside sources). Is that accurate? If so, I'm wondering how to name this appropriately since it's not only restricting downloads. Something like `FILE_PUT_PARALLELISM`?

Reply:
Yeah, that seems to be how it functions. I think `PUT_FILE_PARALLELISM` is probably going to be the preferred name by @jdoliner .

Reply:
@brycemcanally @actgardner yeah `PUT_FILE_PARALLELISM` sounds good to me. `FILE_PUT` gives me a little mental hiccup because of the order.